components/Histogram/components/Slider.module.css

```diff
@@ -0,0 +1,72 @@
+.slider {
+    position: relative;
+}
+
+.slider__track,
+.slider__range {
+    height: 2px;
+}
+
+.slider__track {
+    position: absolute;
+    background-color: var(--histogram-thumb-color-disabled);
+    width: 100%;
+    z-index: 1;
+}
+
+.slider__range {
+    position: absolute;
+    background-color: var(--histogram-thumb-color-active);
+    z-index: 2;
+}
+
+/* Removing the default appearance */
+.thumb,
+.thumb::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    -webkit-tap-highlight-color: transparent;
+}
+
+.thumb {
+    pointer-events: none;
+    position: absolute;
+    height: 0;
+}
+
+.thumb_zindex_3 {
+    z-index: 3;
```

Comment:
it seems to be ok to do just style, as for width :)

Reply:
removed